src/main/java/event.java

```diff
@@ -0,0 +1,19 @@
+public class event extends Task {
```

Comment:
class name should start with a Capital Letter. Hence, it should be "Event" instead of "event". This is important so we do not mix up method/variable names with the Class name. Your IDE will be able to easily change all occurrences of event with Event, so it should be an easy fix.